packages/dataviews/src/components/dataviews/stories/fixtures.tsx

```diff
@@ -737,15 +737,6 @@ export const fields: Field< SpaceObject >[] = [
 			{ value: 'Solar system', label: 'Solar system' },
 			{ value: 'Ice giant', label: 'Ice giant' },
 		],
-		filterBy: {
-			operators: [ 'isAny', 'isNone', 'isAll', 'isNotAll' ],
-		},
-		getValue: ( { item } ) => {
-			return item.categories;
-		},
-		render: ( { item } ) => {
-			return item.categories.join( ',' );
-		},
-		enableSorting: false,
+		type: 'array-text',
```

Comment:
I'm not sure array-text field make sense to me. Can we instead support an "array" type with a secondary property that define the type of each item in the array?

Reply:
On second thought, given that our current use cases only require string values, we probably don't need the item type property at this moment.

For this PR, we could implement the array field type to handle string values exclusively. We can always extend it to support other types later. Does that sound reasonable?

Reply:
I've renamed it to `array` type.